Icro/ViewController/SettingsViewController.swift

```diff
@@ -13,6 +13,7 @@ class SettingsViewController: UIViewController {
     @IBOutlet weak var outterStackView: UIStackView!
     @IBOutlet private weak var blogSetupView: UIView!
     @IBOutlet private weak var blogSetupSwitch: UISwitch!
+    @IBOutlet weak var blogSetupSwitchLabel: UILabel!
```

Comment:
I think all the newly introduced outlets should be `private` properties as well.